Feature/refactor and add tests and coverage (#60)
little-bear-labs/js-libp2p-webrtc: examples/browser-to-server/README.md

# js-libp2p-webrtc Browser to Server

This example leverages the [vite bundler](https://vitejs.dev/) to compile and serve the libp2p code in the browser. You can use other bundlers such as Webpack, but we will not be covering them here.

## Running the Go Server

To run the Go LibP2P WebRTC server:

```shell

> npm run go-libp2p-server

Reviewer: Am I to understand correctly that this command assumes you have go installed already? And at whatever version is necessary? Is it possible npm could make sure that's true for you?

```

> Copy the multiaddress in the output.

Reviewer 2: might be useful to show an example of such output and multiaddress

## Running the Example

In a separate console tab, install dependencies and start the Vite server:

```shell
npm i && npm run start
```

The browser window will automatically open.

> Using the copied multiaddress from the Go server, pasted that value into the `Server MultiAddress` input and click the `Connect` button.

Reviewer 2: ```suggestion
Using the copied multiaddress from the Go server, paste it into the `Server MultiAddress` input and click the `Connect` button.
```

> Once the peer is connected, click the message section will appear.  Enter a message and click the `Send` button.

Reviewer 2: ```suggestion
Once the peer is connected, click the message section once it appeared.  Then enter a message and click the `Send` button.
```
?
